apps/web/src/api/api.client.ts

```diff
@@ -78,46 +59,42 @@ function buildUrl(url: string, absoluteUrl: boolean) {
   return absoluteUrl ? url : `${API_ROOT}${url}`;
 }
 
-function getHeaders() {
-  // TODO: change the way we get the clerk token
-  const token = getToken();
-  const lastEnvironmentId = getEnvironmentId();
-
-  return token
-    ? {
-        Authorization: `Bearer ${token}`,
-        'Novu-Environment-Id': lastEnvironmentId || '',
-      }
-    : {};
-}
-
 // WIP: The static API client needs to be replaced by a dynamic API client where api keys are injected.
 export function buildApiHttpClient({
   baseURL = API_ROOT || 'https://api.novu.co',
   secretKey,
   jwt,
```

Comment:
is this `jwt` param needed here? I don't see it being used in the function

Reply:
Good catch. I removed it.